src/components/MenuSelect.vue

```diff
@@ -17,7 +17,7 @@
           :class="suit('option')"
           value=""
         >
-          {{ label }}
+          <slot name="all">See all</slot>
```

Comment:
The name `all` is confusing we don't know where it's gonna be used. Maybe more something like: `seeAll`, `seeAllLabel`, ...

Reply:
going with `defaultOption`